### PR TITLE
Update to latest Stable Cadence

### PR DESCRIPTION
**Pull request by SupunS**
## Description

Update to
- cadence [v0.41.0-stable-cadence.1](https://github.com/onflow/cadence/releases/tag/v0.41.0-stable-cadence.1)
- flow-go-sdk [v0.44.0-stable-cadence.1](https://github.com/onflow/flow-go-sdk/releases/tag/v0.44.0-stable-cadence.1)
- flow-go [090be4adebb5b70ad1caa6b0e3ca24975440dfde](https://github.com/onflow/flow-go/pull/4663/commits/090be4adebb5b70ad1caa6b0e3ca24975440dfde)
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to GitHub issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-emulator/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the GitHub PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: storage/remote/store_test.go [hunk 1]
```diff
@@ -156,6 +156,10 @@ func (a testClient) ListSealsForHeight(ctx context.Context, in *archive.ListSeal
 }
 
 func Test_SimulatedMainnetTransaction(t *testing.T) {
+
+	// TODO: enable
+	t.SkipNow()
```
Comment (SupunS) [inline storage/remote/store_test.go:161]: This test and below, rely on a mainnet contract, that is not compatible with stable cadence.

File: emulator/contracts_test.go [hunk 2]
```diff
@@ -28,6 +28,9 @@ import (
 
 func TestCommonContractsDeployment(t *testing.T) {
 
+	// TODO: enable
+	t.SkipNow()
```
Comment (SupunS) [inline emulator/contracts_test.go:32]: This test relies on `flow-nft/ExampleNFT` contract that is not compatible with stable-cadence. Not sure whats the updated version for `flow-nft` is.

Comment (joshuahannan) [reply emulator/contracts_test.go:32]: I got the `ExampleNFT-V2` deploying in this branch: https://github.com/onflow/flow-nft/pull/126
So you can use that now if that is what you need

Comment (SupunS) [reply emulator/contracts_test.go:32]: This test is still failing, now for a different reason: `ExampleNFT-V2` now imports `MultipleNFT`, so `MultipleNFT` is also needed to be pre-deployed to the emulator before the test. Not sure how to do that. So kept this disabled for now.

File: emulator/contracts.go [hunk 3]
```diff
@@ -25,34 +25,34 @@ import (
...
-			Source:      nftstorefront.NFTStorefront(2, ftAddress.String(), serviceAddress.String()),
-		},
-		{
-			Name:        "NFTStorefront",
-			Address:     serviceAddress,
-			Description: "✨  NFT Storefront contract",
-			Source:      nftstorefront.NFTStorefront(1, ftAddress.String(), serviceAddress.String()),
+			Source:      contracts.ExampleNFT(serviceAddress, serviceAddress, serviceAddress, serviceAddress),
 		},
+		//{
```
Comment (SupunS) [inline emulator/contracts.go:44]: Disabled these two for now, since `NFTStorefront` is not yet migrated.

**Review by turbolent — APPROVED** [review]

**Pull request merged**